packages/docs/getting-started/running-a-validator.md

```diff
@@ -119,8 +119,8 @@ $ celocli account:unlock --account $CELO_VALIDATOR_ADDRESS --password <YOUR_SECO
 In a new tab, make a locked Gold account for both of your addresses by running the Celo CLI. This will allow you to stake Celo Gold, which is required to register a validator and validator groups:
 
 ```
-$ celocli lockedgold:register --from $CELO_VALIDATOR_GROUP_ADDRESS
-$ celocli lockedgold:register --from $CELO_VALIDATOR_ADDRESS
+$ celocli account:register --from $CELO_VALIDATOR_GROUP_ADDRESS --name <GROUP_NAME_OF_YOUR_CHOICE>
```

Comment:
Is this what celo.org/docs reads from? If so, we probably don't want to update this until we push a new version of the CLI

Reply:
It does not, so my understanding is that we make docs changes with the PRs and then at some point deploy them